docs/source/get_involved/governance.rst

```diff
@@ -200,17 +217,23 @@ Becoming inactive in the above sense means not contributing for the period via:
 * commenting on pull requests or issues
 * attending one of the regular meetings
 
-Becoming active (after becoming inactive) in the above sense requires one of:
+of a scope that goes beyond algorithm maintainer expectations - i.e., PR or issues related to owned algorithms do not count.
+
+Becoming active (after becoming inactive) in the above sense requires:
 
-* an approved pull request authored by the core developer
-* a contribution to the community that is minuted in one of the regular meetings
+* a contribution within the last month that goes beyond algorithm maintainer expectations, e.g.:
+  * an approved pull request authored by the core developer
+  * a substantial contribution to the community that is minuted in one of the regular meetings
+* success via the same voting procedure as for new core developers, with the exception that former core developers can also self-nominate
 
 .. _coc-committee-members:
 
 CoC committee members
 ~~~~~~~~~~~~~~~~~~~~~
 
-CoC members are contributors with special rights and responsibilities.
+CoC committee members are contributors with special rights and responsibilities.
```

Comment:
```suggestion
CoC committee members are contributors with special rights and responsibilities. CoC member should not be members of CC due to conflicts of interest.
```

Reply:
adopting that rule would, at current, result in either a CC with 2 members (below numfocus threshold) or an empty CoC.

I'm happy with this proposed rule, but let's make it a condition for appointment instead, in the "appointment" paragraph, so current members (i.e., me) are not automatically removed from either committee.

I'm also happy to sharpen the rule as soon as it seems we are in a state where we have enough members for CoC committee and CC, and I feel that the CoC is operating properly (i.e., addressing problematic issues by the book when they occur).

Reply:
accepted (moved to expectations, because it is a "should")

Reply:
Did I propose the "should", if yes it was my mistake. I think it cant be a should, a should is meaningless bcz it means its an optional requirement. It has to be a must, otherwise there is no sense to have it here.